src/qibolab/serialize.py

```diff
@@ -91,10 +93,19 @@ def load_instrument_settings(runcard: dict, instruments: InstrumentMap) -> Instr
 
 def dump_qubits(qubits: QubitMap, pairs: QubitPairMap, couplers: CouplerMap = None) -> dict:
     """Dump qubit and pair objects to a dictionary following the runcard format."""
-    native_gates = {
-        "single_qubit": {q: qubit.native_gates.raw for q, qubit in qubits.items()},
-        "two_qubit": {},
-    }
+
+    if couplers:
+        native_gates = {
+            "single_qubit": {q: qubit.native_gates.raw for q, qubit in qubits.items()},
+            "coupler": {c: coupler.native_pulse.raw for c, coupler in couplers.items()},
+            "two_qubit": {},
+        }
+    else:
+        native_gates = {
+            "single_qubit": {q: qubit.native_gates.raw for q, qubit in qubits.items()},
+            "two_qubit": {},
+        }
```

Comment:
Maybe
```suggestion 
    native_gates = {"single_qubit": {q: qubit.native_gates.raw for q, qubit in qubits.items()}}
    if couplers is not None:
            native_gates["coupler"] = {c: coupler.native_pulse.raw for c, coupler in couplers.items()}
    native_gates["two_qubit"] = {}
```
is better because it has less duplication.

Reply:
The `is not None` was causing problem, as you asked me to define couplers by a empty dict I need to use` if couplers:` for it no to get into the loop.